Marlin/Configuration_adv.h

```diff
@@ -1268,6 +1268,13 @@
   #define USER_GCODE_5 "G28\nM503"
 #endif
 
+/**
+ * Specify an action command to send to the host when the printer is killed.
+ * Will be sent in the form '//action:KILL_ACTION', e.g. '//action:poweroff'.
+ * The host must be configured to handle the action command.
+ */
+//#define KILL_ACTION "poweroff"
```

Comment:
Will do - wanted to have the conversation about the parameter name before I went and edited a bunch of files.  ACTION_AT_KILL seems good to me.  Do you have a preference?